src/components/Select/Select.stories.tsx

```diff
@@ -165,18 +165,76 @@ const selectCat: StoryObj['play'] = async (playOptions) => {
 };
 
 export const Default: StoryObj = {
+  parameters: {
+    docs: {
+      source: {
+        code: `
+// props: {aria-label: 'Favorite Animal'}
+function InteractiveExampleUsingChildren(props: Props) {
+  const { variant, optionsAlign, optionsClassName, disabled, value } = props;
+  const compact = variant === 'compact';
+
+  const [selectedOption, setSelectedOption] = React.useState<
+    SelectOption | undefined
+  >(value);
+
+  return (
+    <div className="mb-10 p-8">
+      <Select
+        aria-label={props['aria-label']}
+        className={clsx(!compact && 'w-60')}
+        data-testid="dropdown"
+        disabled={disabled}
+        onChange={setSelectedOption}
+        optionsAlign={optionsAlign}
+        optionsClassName={optionsClassName}
+        value={selectedOption}
+        variant={variant}
+      >
+        {props.labelComponent}
+        <Select.Button>{selectedOption?.label || 'Select'}</Select.Button>
+        <Select.Options>
+          {exampleOptions.map((option) => (
+            <Select.Option key={option.key} value={option}>
+              {option.label}
+            </Select.Option>
+          ))}
+        </Select.Options>
+      </Select>
+    </div>
+  );
+}`,
+      },
+    },
+  },
   render: () => (
     <InteractiveExampleUsingChildren aria-label="Favorite Animal" />
   ),
 };
 
 export const Disabled: StoryObj = {
+  parameters: {
+    docs: {
+      source: {
+        code: '// refer to "Default" story code',
+      },
+    },
+  },
   render: () => (
     <InteractiveExampleUsingChildren aria-label="Favorite Animal" disabled />
   ),
 };
 
 export const DefaultWithVisibleLabel: StoryObj = {
+  parameters: {
+    docs: {
+      source: {
+        code: `
+// refer to "Default" story code with:
+// props: {aria-label: 'Favorite Animal', labelComponent: <Select.Label>Favorite Animal</Select.Label>}`,
```

Comment:
An example where props is explicitly stated here to show usage of the `labelComponent` prop, but if this story were to change in the future, this might also need to be updated